web-marketplace/src/components/organisms/UserSellOrders/UserSellOrders.Toolbar.tsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from 'react';
+import { msg, Trans } from '@lingui/macro';
+import { useLingui } from '@lingui/react';
+
+import ContainedButton from 'web-components/src/components/buttons/ContainedButton';
+import { Subtitle } from 'web-components/src/components/typography';
+import { cn } from 'web-components/src/utils/styles/cn';
+
+import { CreateSellOrderFlow } from 'features/marketplace/CreateSellOrderFlow/CreateSellOrderFlow';
+import { useFetchEcocredits } from 'pages/Dashboard/MyEcocredits/hooks/useFetchEcocredits';
+
+interface UserSellOrdersToolbarProps {
+  wrapperClassName?: string;
+  refetchSellOrders: () => void;
+}
+
+export const UserSellOrdersToolbar = ({
+  wrapperClassName,
+  refetchSellOrders,
+}: UserSellOrdersToolbarProps) => {
+  const { _ } = useLingui();
+  const [isSellFlowStarted, setIsSellFlowStarted] = useState(false);
+  const { credits } = useFetchEcocredits({});
```

Comment:
right now, we only get the first page of credits, I think you should set `isPaginatedQuery: false` param